### PR TITLE
Milestone 4: Code review pull request

### PR DESCRIPTION
**Pull request by HussenYesufAlli**
In this milestone I:
Add index.html 
Add styles.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
+    <div class="container">
+    <head>
+        
+        <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
+    </head>    
+    <link rel="stylesheet" href="styles.css">
```
Comment (HussenYesufAlli) [inline index.html:9]: I can see the head tag is inside the body section. Please move up before the body section including the style link and keep the header tag in the body there. In addition, please add a title that expresses your project page. The contents of a page title can have significant implications for search engine optimization.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  margin-bottom: 0.5rem;
+}
+
+/* .form-lebels {
+  display: flex;
+  align-items: center;
+  font-size: 1.125rem;
+  margin-bottom: 0.5rem;
+} */
+
```
Comment (HussenYesufAlli) [inline styles.css:33]: Please do not leave unused code as a comment in your codebase. unused code as a comment makes your code not clean and bulky.

**Review by HussenYesufAlli — COMMENTED** [review]
Good job so far! 👍
There are some issues that you still need to change, please check my comments below
Thanks 😁 

Required Changes ♻️

**Review by Gambit142 — APPROVED** [review]
Hi @HussenYesufAlli 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Good use of inline comment ✔️ 
- Comments are polite ✔️ 
- Comments are easy to understand ✔️ 
- Detailed reasons for suggesting changes were given ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**